app/controllers/users/sessions_controller.rb

```diff
@@ -29,10 +29,15 @@ def new
     def create
       track_authentication_attempt(auth_params[:email])
 
+      clear_session_bad_password_count_if_window_expired
```

Comment:
would this be a good candidate for a `before_filter`? Or would that just make it harder to find?

Reply:
Great idea!